Code/ThirdParty/glfw/finalize.cmake

```diff
@@ -11,4 +11,10 @@ if(TARGET glfw)
 
 
     set_property(GLOBAL APPEND PROPERTY "EXPORT_PROJECTS" glfw)
+    
+    ez_set_project_ide_folder(glfw "Code/ThirdParty/GLFW")
+    ez_set_project_ide_folder(update_mappings "Code/ThirdParty/GLFW")

```

Comment:
what is this ?

Reply:
GLFW creates two cmake targets. One called "glfw" and the other one called "update_mappings" (which generates some lookup tables). I move both of them into the Code/ThirdParty filter within the visual studio solution.